extensions/amp-sidebar/1.0/amp-sidebar.css

```diff
@@ -53,10 +53,10 @@
    z-index: 2147483646;
    width: 100%;
    position: fixed;
-   top: 0;
```

Comment:
That is fine , it is up to the developer to adjust this , but by default it should be 0.